site/app/controllers/PollController.php

```diff
@@ -329,6 +338,12 @@ public function submitEdits() {
                 new RedirectResponse($this->core->buildCourseUrl(['polls']))
             );
         }
+        $file_path = null;
+        if (isset($_FILES['image_file']) && $_FILES["image_file"]["name"] !== "") {
+            $file = $_FILES["image_file"];
+            copy($file["tmp_name"], $file_path);
+            $file_path = FileUtils::joinPaths($this->core->getConfig()->getCoursePath(), "uploads", "polls", "poll_image_" . $_POST["poll_id"] . "_" . $_FILES["name"]);
+        }
```

Comment:
This method should probably be modified to remove the old poll image, unless we want to leave these files on the filesystem for some reason (though not reachable beyond ssh)?